code/ATMOSPHERICS/components/unary_devices/vent_pump.dm

```diff
@@ -3,6 +3,8 @@
 #define PRESSURE_CHECKS 1
 #undefine
 
+var/global/list/all_vent_pumps = list()
```

Comment:
this should be in https://github.com/ParadiseSS13/Paradise/blob/master/code/_globalvars/lists/objects.dm regardless